bullets/enemy.py

```diff
@@ -1,5 +1,28 @@
 from .actor import Actor
 
-
 class Enemy(Actor):
-    pass
+    def __init__(maxhealth, patterns, pcb1, pcb2, pcb3 )
```

Comment:
* the first parameter of a method that uses `self` should be `self`
* `def` lines should end with a `:`